src/components/Header/Header.scss

```diff
@@ -0,0 +1,45 @@
+.head {
+  align-items: center;
+  background: #f5f6f7;
```

Comment:
Should be white (except for the menu thingy on the left, which we can add later)